find_xml_file.rb

```diff
@@ -0,0 +1,15 @@
+
+def find_xml_file(folder)
+  files = Dir.entries(folder)
+
+  xml_file = nil
```

Comment:
Is there a chance there might be more than 1 .xml file in the directory being searched?  if so turning nil into an array might be helpful.